oct/ansible/oct/roles/dependencies/tasks/register_repositories.yml

```diff
@@ -34,12 +34,13 @@
     name: https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
     state: present
 
-- name: Add RHEL 7 Server Ansible 2.4 Repository
+- name: Add RHEL 7 Server Ansible Repositories
   yum_repository:
-    name: 'rhel-7-server-ansible-2.4-rpms'
+    name: "rhel-7-server-ansible-{{ item }}-rpms"
     state: present
-    description: 'RHEL 7 Server Ansible Repository'
-    baseurl: 'https://mirror.openshift.com/enterprise/rhel/rhel-7-server-ansible-2.4-rpms/'
+    description: "RHEL 7 Server Ansible {{ item }} Repository"
+    baseurl: "https://mirror.openshift.com/enterprise/rhel/rhel-7-server-ansible-{{ item }}-rpms/"
+    enabled: "{{ 'yes' if item == '2.4' else 'no' }}"
```

Comment:
If this does what you mean for it to do - only enable the 2.4 repo - then this lgtm

Reply:
Yes, my intent was to just leave 2.4 repo enabled as it was before and everything else would be enabled in a job that requires it. BTW, I updated the description with and alternative approach.